benchmarks/data/tardis_configv1_verysimple.yml

```diff
@@ -0,0 +1,49 @@
+tardis_config_version: v1.0
+
+supernova:
+    luminosity_requested: 2.8e9 solLum
+    time_explosion: 13 day
+
+atom_data: kurucz_atom_pure_simple.h5
+
```

Comment:
Why are we duplicating the name of a config file? I would suggest calling this "benchmark" instead of "verysimple"